### PR TITLE
[DR-3444][DR-3445][DR-3446][DR-3447] UI Quality of life fixes

### PR DESCRIPTION
**Pull request by nmalfroy**
Combined into a single PR that should be reviewed commit by commit:
- Fixed issue where some images weren't loading
- Fixed the footer on the login page that was scrolling off of the screen
- Pointed to the new faster DUOS endpoint
- Also made it so that the snapshot page wouldn't lock if the duos endpoint was slower
- Show job durations!
- Add a refresh button on the datasets/snapshots/activities page!

image

You all can see it in actin here:
https://jade-nm.datarepo-dev.broadinstitute.org/

### CODE REVIEW

File: src/components/table/JobTable.tsx [hunk 1]
```diff
@@ -123,6 +125,19 @@ const JobTable = withStyles(styles)(
           </span>
         ),
       },
+      {
+        label: 'Duration',
+        name: 'job_duration',
+        allowSort: false,
+        width: '10%',
```
Comment (okotsopoulos) [inline src/components/table/JobTable.tsx:138]: Heads up that as the job duration exceeds an hour, this is liable to be cut off.  

Screenshot 2024-02-06 at 4 53 59 PM

**Review by fboulnois — APPROVED** [review]
👍 on DUOS changes

**Review by okotsopoulos — APPROVED** [review]
Such a nice collection of little improvements!  I can tell you had a good time with this one :) loading the snapshot summary page is super snappy now, that's going to help everyone.

I have a nit on longer durations getting cut off but not a blocker to merge.

**Comment by nmalfroy** [comment]
> Such a nice collection of little improvements! I can tell you had a good time with this one :) loading the snapshot summary page is super snappy now, that's going to help everyone.
> 
> I have a nit on longer durations getting cut off but not a blocker to merge.

Good call.  I added ellipses and tooltips on the jobs table to handle when the text gets cut off

**Pull request merged**